test/test_connection.py

```diff
@@ -153,6 +150,14 @@ def test_ping_pong(self, as_client):
             data = wire_data[2:]
         assert data == payload
 
+        # Verify that connection vends the PongReceive event with the correct payload
```

Comment:
Oxford dictionary says:
> to vend: Offer (small items) for sale, either from a stall or from a slot machine.

I guess it would be a valid use here, but this is a rather uncommon verb to use.
👍  for english skillz.
👎  for easily readable documentation.